docs/content/develop/field-reference.md

```diff
@@ -335,6 +338,30 @@ item_type: !ruby/object:Api::Type::NestedObject
         MULTI_LINE_FIELD_DESCRIPTION
 ```
 
+### `item_validation`
+Array only. Controls the [`ValidateFunc`](https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors#validatefunc)
```

Comment:
We should add something like:

> For arrays of enums, this will override the enum validation. If you need additional validation on top of an enum, ensure that the supplied validation func also verifies the enum values are correct.

Reply:
done